src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+import Big from 'big.js';
+
+function operate(numberOne, numberTwo, operation) {
+  const num1 = Big(numberOne);
+  const num2 = numberTwo ? Big(numberTwo) : null;
```

Comment:
I am still trying the motive behind this line🤔.

Reply:
Thought Big will throw an error is passed null. Thank you very much